javascript-light-bulb/main.js

```diff
@@ -0,0 +1,24 @@
+// Classes that need to be changed:
+//    .background-on/.background-off
+//    .on/.off
+// Start with the bulb in the on position
+// If the bulb is on when you click on it, change the circle and container class
+// names to the off designations.
+// Otherwise, if the bulb is off when you click it, change the circle and container class names
+// to the on designations.
+// Add an event listener for the click.
+
+var $circle = document.querySelector('.circle');
+var $container = document.querySelector('.container');
+
+function onOff(event) {
+  if ($circle.className === 'circle on') {
```

Comment:
By doing it this way, the action you take in your JavaScript is dependent on checking the current state of the element, which means your JavaScript is reliant on information from another language to know what to do. Instead, you should make a variable to keep track of if the light is currently on or off, and use that variable's value to determine what action to take on the DOM